### PR TITLE
Upgrading major deps 2

### PR DESCRIPTION
**Pull request by piotr-s-brainhub**
Deployed to https://beghp.github.io/gh-pages-rc-1<br><br>- [ ] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)<br>- [ ] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`<br><br>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,19 +41,20 @@
...
     "classnames": "^2.2.6",
     "css-loader": "^3.4.2",
-    "enzyme": "^2.9.1",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
     "eslint": "^6.8.0",
     "eslint-config-brainhub": "^1.12.0",
     "eslint-watch": "^6.0.1",
-    "extract-text-webpack-plugin": "^2.1.2",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```
Comment (RobertHebel) [inline package.json]: The whole PR looks good for me except using libraries in beta version what can be risky. Personally I believe we should use the latest stable release

Comment (piotr-s-brainhub) [reply package.json]: @RobertHebel 

the reason of using beta versions is that the latest version of one library requires a beta version of another:
- the latest `enzyme` requires beta `chai-enzyme`
- if I remember correctly, the latest `webpack` requires beta `extract-text-webpack-plugin`

Comment (piotr-s-brainhub) [reply package.json]: as they are dev (not prod) dependencies and beta (not alpha), IMO we can take a risk

Comment (RobertHebel) [reply package.json]: that it's fine by me

**Pull request merged**